app/api/sessions.py

```diff
@@ -49,22 +49,23 @@ def validate_date(self, data, original_data):
 
     id = fields.Str(dump_only=True)
     title = fields.Str(required=True)
-    subtitle = fields.Str()
-    level = fields.Int()
-    short_abstract = fields.Str()
-    long_abstract = fields.Str()
-    comments = fields.Str()
+    subtitle = fields.Str(allow_none=True)
+    level = fields.Int(allow_none=True)
+    short_abstract = fields.Str(allow_none=True)
+    long_abstract = fields.Str(allow_none=True)
+    comments = fields.Str(allow_none=True)
     starts_at = fields.DateTime(required=True)
     ends_at = fields.DateTime(required=True)
-    language = fields.Str()
-    slides_url = fields.Url()
-    video_url = fields.Url()
-    audio_url = fields.Url()
-    signup_url = fields.Url()
-    state = fields.Str(validate=validate.OneOf(choices=["pending", "accepted", "confirmed", "rejected", "draft"]))
+    language = fields.Str(allow_none=True)
+    slides_url = fields.Url(allow_none=True)
+    video_url = fields.Url(allow_none=True)
+    audio_url = fields.Url(allow_none=True)
+    signup_url = fields.Url(allow_none=True)
+    state = fields.Str(validate=validate.OneOf(choices=["pending", "accepted", "confirmed", "rejected", "draft"]),
+                       allow_none=True)
     created_at = fields.DateTime(dump_only=True)
     deleted_at = fields.DateTime(dump_only=True)
-    submitted_at = fields.DateTime()
+    submitted_at = fields.DateTime(allow_none=True)
     is_mail_sent = fields.Boolean()
     microlocation = Relationship(attribute='microlocation',
                                  self_view='v1.session_microlocation',
```

Comment:
cant there be some cases when the location for a session be null?
like an online session or something.